source/utilities.cc

```diff
@@ -94,27 +94,37 @@ namespace aspect
                                const std::vector<std::string> &input_field_names,
```

Comment:
For some reason, the names of the first two function arguments are different in the declaration and in the implementation here.